### PR TITLE
MVG popup icon spiner

### PR DESCRIPTION
**Pull request by AndrewRAM33**
MVG field open icon inactive without popup widget rowMeta (#299)

Checking popup widget rowMeta from mvg field. While rowMeta dont receive the icon open popup changed to spiner.

### CODE REVIEW

File: src/components/ui/PickInput/PickInput.tsx [hunk 1]
```diff
@@ -2,14 +2,22 @@ import React from 'react'
...
 
-export interface PickInputProps {
+export interface PickInputOwnProps {
     disabled?: boolean,
     value?: string,
     onClick?: () => void,
     onClear?: () => void,
     className?: string,
     placeholder?:  string
+    popupBcName?: string, // TODO: not optional in 2.0
```
Comment (Dergash) [inline src/components/ui/PickInput/PickInput.tsx:16]: I think an optional `spin?: boolean` will work better here as `PickInput` component will stay a simple ui control that way.

File: src/components/ui/Multivalue/MultivalueTag.tsx [hunk 2]
```diff
@@ -4,19 +4,27 @@ import {MultivalueSingleValue} from '../../../interfaces/data'
 import {MultivalueFieldMeta} from '../../../interfaces/widget'
 import styles from './MultivalueTag.less'
 import cn from 'classnames'
+import {Store} from '../../../interfaces/store'
+import {connect} from 'react-redux'
+import {buildBcUrl} from '../../..'
```
Comment (Dergash) [inline src/components/ui/Multivalue/MultivalueTag.tsx:9]:
```tsx
import {buildBcUrl} from '../../../utils/strings'
```

File: src/components/ui/Multivalue/MultivalueTag.tsx [hunk 3]
```diff
@@ -4,19 +4,27 @@ import {MultivalueSingleValue} from '../../../interfaces/data'
...
 
-export interface MultivalueTagProps {
+export interface MultivalueTagOwnProps {
     disabled: boolean,
     placeholder?: string,
     value: MultivalueSingleValue[],
     widgetFieldMeta: MultivalueFieldMeta,
     bcName: string,
+    popupBcName: string,
```
Comment (Dergash) [inline src/components/ui/Multivalue/MultivalueTag.tsx:17]: I think it's better to make an optional `loading?: boolean` props and leave the component unconnected while calculating the flag in [MultivalueField](https://github.com/tesler-platform/tesler-ui/blob/master/src/components/Multivalue/MultivalueField.tsx)

File: src/components/ui/Multivalue/MultivalueTag.tsx [hunk 4]
```diff
@@ -61,17 +69,37 @@ const MultivalueTag: React.FunctionComponent<MultivalueTagProps> = (props) => {
...
-            <div
-                className={cn(
-                    styles.iconContainer,
-                    {[styles.disabled]: props.disabled}
-                )}
-                onClick={handleOpen}
-            >
-                <Icon type="folder-open"/>
-            </div>
+            {props.popupRowMetaDone || !props.popupBcName
```
Comment (Dergash) [inline src/components/ui/Multivalue/MultivalueTag.tsx:72]:
```tsx
const loading = props.popupRowMetaDone || !props.popupBcName
// ...
<div className={cn(styles.iconContainer, { [styles.disabled]: props.disabled })}>
    <Icon
        type={loading ? 'folder-open' : 'loading'}
        spin={loading}
        onClick={loading ? handleOpen : undefined}
    />
</div>
```

File: src/components/ui/PickInput/PickInput.tsx (hunk 1 above)
Comment (AndrewRAM33) [reply src/components/ui/PickInput/PickInput.tsx]: fixed as `loading?: boolean` option

File: src/components/ui/Multivalue/MultivalueTag.tsx (hunk 2 above)
Comment (AndrewRAM33) [reply src/components/ui/Multivalue/MultivalueTag.tsx]: fixed

File: src/components/ui/Multivalue/MultivalueTag.tsx (hunk 3 above)
Comment (AndrewRAM33) [reply src/components/ui/Multivalue/MultivalueTag.tsx]: fixed

File: src/components/ui/Multivalue/MultivalueTag.tsx (hunk 4 above)
Comment (AndrewRAM33) [reply src/components/ui/Multivalue/MultivalueTag.tsx]: fixed with inverted logic of prop `loading`

**Pull request merged**